diff -urN smtube-1.6/Makefile smtube-1.6-patched/Makefile
--- smtube-1.6/Makefile	2012-02-27 10:43:50.000000000 +1100
+++ smtube-1.6-patched/Makefile	2013-03-26 18:26:18.591110747 +1100
@@ -21,7 +21,7 @@
 
 src/smtube:
 #	./get_svn_revision.sh
-	+cd src && $(QMAKE) $(QMAKE_OPTS) && $(DEFS) make
+	+cd src && $(QMAKE) "QMAKE_CFLAGS+=$(CFLAGS)" "QMAKE_CXXFLAGS+=$(CXXFLAGS)" "QMAKE_CXXFLAGS_RELEASE=" "QMAKE_LFLAGS+=$(LDFLAGS)" $(QMAKE_OPTS) && $(DEFS) make
 	cd src && $(LRELEASE) smtube.pro
 
 clean:
